packages/js/tests/integrations-page/GoogleSiteKitIntegration.test.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import { render, screen } from "../test-utils";
+import { GoogleSiteKitIntegration } from "../../src/integrations-page/google-site-kit-integration";
+import { expect } from "@jest/globals";
+
+describe( "GoogleSiteKitIntegration", () => {
```

Comment:
These are the happy paths.
I wonder, with all that logic if/else, what would happen if you brute force all the combinations 😁